packages/python-packages/api-stub-generator/apistub/_apiview.py

```diff
@@ -68,12 +69,37 @@ def add_whitespace(self):
     def add_space(self):
         self.add_token(Token(" ", TokenKind.Whitespace))
 
-    def add_new_line(self, additional_line_count=0):
-        self.add_token(Token("", TokenKind.Newline))
-        for n in range(additional_line_count):
-            self.add_space()
+    def add_newline(self, *, additional_line_count=0):
```

Comment:
additional_line_count is not used.

Reply:
Thanks for the catch. Now "add_newline" cannot add blank lines, so this is intentional. I'll remove it from the signature.

Reply:
Removed.